bundles/ui/org.eclipse.smarthome.ui/META-INF/MANIFEST.MF

```diff
@@ -18,6 +18,7 @@ Import-Package: javax.imageio,
  org.eclipse.jetty.client.api,
  org.eclipse.jetty.client.util,
  org.eclipse.jetty.http,
+ org.eclipse.jetty.proxy,
```

Comment:
We need to have it in the TP in any case, otherwise the code won't compile, so I will also have to add it to the OH2 IDE TP.
The "optional=true" is relevant at runtime: with it, the smarthome.ui bundle can be resolved and used even if jetty-proxy bundle is NOT available (and I guess on a system which only has servlet 2.4, it simply CANNOT be available anyhow).